### PR TITLE
Create Interview Information Sheet, #620

### PR DESCRIPTION
**Pull request by levesswe**
Blip: This is a script that produces a quick-reference of generic and case-specific information intended to be used during an interview with a client to assist in answering client questions.

### CODE REVIEW

File: actions/interview-information-sheet.vbs [hunk 1]
```diff
@@ -0,0 +1,805 @@
...
+
+
+'If the participant selected by the user is NCP, then the script completes the form fields for NCP using information produced by using the doc_info class methods. 
+'On line 731, we named our doc_info class object, "info". To efficiently obtain case information, we call methods for the property information of our doc_info object.
+'To call, start with "info", then a period, and then the name of the property.  
+'To review the properties you can use, check out the class definition in lines 69-684.
+'Unless classes are added to the functions library, it is necessary to have the class definition in your script if you want to use it.
+IF participant = "NCP" Then	
+	set objDoc = objWord.Documents.Add(word_documents_folder_path & "interview-information-sheet.docx")
+		With objDoc
```
Comment (theVKC) [inline actions/interview-information-sheet.vbs]: The indent here is one too many. No big deal, but for the future, can you not indent the "with" after the "set"?

File: actions/interview-information-sheet.vbs [hunk 2]
```diff
@@ -0,0 +1,805 @@
...
+			.FormFields("cch").Result = FormatCurrency (Ccur(info.cch_amount))
+			.FormFields("cms").Result = FormatCurrency(CCur(info.cms_amount))
+			.FormFields("cmi").Result = FormatCurrency(CCur(info.cmi_amount))
+			.FormFields("ccc").Result = FormatCurrency(CCur(info.ccc_amount))
+			.FormFields("csp").Result = FormatCurrency(CCur(info.csp_amount))
+			.FormFields("arrears_payback").Result = FormatCurrency(info.monthly_non_accrual)
+			.FormFields("ncp_mci").Result = info.ncp_mci	
+	End With
+END IF
+
```
Comment (theVKC) [inline actions/interview-information-sheet.vbs]: Needs a "script_end_procedure" at the end. That sends data to the usage log for counties using it.

File: actions/interview-information-sheet.vbs [hunk 3]
```diff
@@ -0,0 +1,805 @@
...
+  ListBox 125, 10, 60, 15, "NCP"+chr(9)+"CP", participant
+EndDialog
+
+
+'------Start of Class definitions--------------------------------------------------------------------------------
+'>>>>> CLASSES!!!!!!!!!!!!!!!!!!!!! <<<<<
+' This CLASS contains properties used to populate documents
+' These properties should not be used for other applications in scripts.
+' Every time you call the property, the script will use the class definition to efficiently obtain the requested information.
+CLASS doc_info
```
Comment (theVKC) [inline actions/interview-information-sheet.vbs]: EXCELLENT documentation here. Well done.

**Comment by levesswe** [comment]
:P How did I miss that script_end_procedure???!!! It’s fixed now…

Wendy LeVesseur
Child Support Officer

Anoka County Government Center
2100 3rd Avenue, Suite 401
Anoka, MN 55303
763-422-7323

**Comment by theVKC** [comment]
I'm in the middle of a big documentation push. As soon as this is done, I will be testing your work. :)

**Comment by theVKC** [comment]
I did not forget about this, @levesswe. Hoping to wrap up some other things and then I'll be able to fully test this.

**Comment by levesswe** [comment]
LOL, I was starting to wonder!

**Comment by theVKC** [comment]
Planning on testing this tomorrow, or today. Hoo boy.

**Pull request merged**